docs/articles_en/about-openvino/contributing/code-contribution-guide.rst

```diff
@@ -0,0 +1,88 @@
+   requests. The `“Good First Issue” <https://github.com/orgs/openvinotoolkit/projects/3>`__
+   board is a good place to start. If you have a new idea for the contribution,
+   make sure to first create a proper issue, discussion, or feature request.
+
+   Here are some of the components you may choose to work on.
+
+   .. tab-set::
+
+      .. tab-item:: APIs
+
+         - `Core C++ API <https://github.com/openvinotoolkit/openvino/tree/master/src/core>`__
+         - `C API <https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/c>`__
+         - `Python API <https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/python>`__
+         - `JavaScript (Node.js) API <https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/js>`__
+
+      .. tab-item:: Frontends
+
+         - `IR Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/ir>`__
+         - `ONNX Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx>`__
+         - `PaddlePaddle Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/paddle>`__
+         - `PyTorch Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/pytorch>`__
+         - `TensorFlow Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow>`__
+         - `TensorFlow Lite Frontend <https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow_lite>`__
+
+      .. tab-item:: Plugins
+
+         - `Auto plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto>`__
+         - `CPU plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu>`__
+         - `GPU plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu>`__
+         - `NPU plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu>`__
+         - `Hetero plugin <https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/hetero>`__
+         - `Template plugin <https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/template>`__
+
+      .. tab-item:: Tools
+
+         - `Benchmark Tool <https://github.com/openvinotoolkit/openvino/tree/master/tools/benchmark_tool>`__
+         - `Model Conversion <https://github.com/openvinotoolkit/openvino/tree/master/tools/ovc>`__
+
+2. **Assign yourself to the issue.**
+
+   To get assigned to a task, simply leave a comment with the ``.take`` command in
+   the selected issue. You can always ask OpenVINO developers for guidance,
+   both technical and organizational:
+
+   - assign users in the **“Contact points”** section,
+   - visit `Intel DevHub Discord server <https://discord.gg/7pVRxUwdWG>`__ to ask
+     questions in the channel dedicated to **“Good First Issue”** support, or any other.
+
+3. **Build OpenVINO.**
```

Comment:
This point could be renamed to "Follow technical contributing guide" and point to https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md#Technical-guide.

Reply:
This article is based on the "technical guide" and reference to the source article in this case is not needed.